MIGRATION.md

```diff
@@ -721,8 +721,8 @@ It has been replaced with a tags-based system which is more flexible than before
 
 `docs.autodocs` takes three values:
 
-- `true`: generate autodocs for every component
-- `false`: don't generate autodocs at all
+- `true`: generate autodocs for every component.
+- `false`: don't generate autodocs at all.
```

Comment:
@shaymolcho, can you revert this and remove the period from the item below (i.e., `tag`) to ensure consistency? The reasoning behind this is that these types of items often don't have any punctuation in the documentation.